sdk/appconfiguration/app-configuration/samples-dev/featureFlag.ts

```diff
@@ -22,124 +21,120 @@ dotenv.config();
-          name: "Microsoft.Targeting",
-          parameters: {
-            audience: {
-              groups: [
-                { name: "group-1", rolloutPercentage: 25 },
-                { name: "group-2", rolloutPercentage: 45 }
-              ],
-              users: ["userA", "userB"],
-              defaultRolloutPercentage: 40
-            }
-          }
-        },
-        { name: "Microsoft.Percentage", parameters: { value: 25 } }
-      ]
-    },
-    enabled: false,
+  const originalFeatureFlag: ConfigurationSetting<FeatureFlagValue> = {
+    key: `sample-feature-flag`,
     isReadOnly: false,
-    key: `${featureFlagPrefix}new-feature-flag-${Math.ceil(100 + Math.random() * 900)}`,
     contentType: featureFlagContentType,
-    description: "I'm a description"
+    value: {
+      enabled: false,
+      description: "I'm a description",
+      conditions: {
+        clientFilters: [
+          // {
+          // // Time window filter - Use this filter to activate the feature for a time period
+          //   name: "Microsoft.TimeWindow",
+          //   parameters: {
+          //     Start: "Wed, 01 May 2021 13:59:59 GMT",
+          //     End: "Mon, 01 July 2022 00:00:00 GMT"
+          //   }
+          // },
+          {
+            // Targeting filter - you can target users/groups of users using this filter
+            name: "Microsoft.Targeting",
+            parameters: {
+              Audience: {
+                Groups: [{ Name: "contoso.com", RolloutPercentage: 50 }], // The feature flag is enabled for 50% of other users in the contoso.com group, because contoso.com is listed in the Groups section with a Percentage of 50.
+                Users: ["test@contoso.com"], // The feature flag is always enabled for user test@contoso.com, because test@contoso.com is listed in the Users section.
+                DefaultRolloutPercentage: 0 // The feature is always disabled for all other users, because the Default percentage is set to 0.
+              }
+              // You can create additional users with @contoso.com email addresses to see the behavior of the group settings. 50% of these users will see the Beta item. The other 50% won't see the Beta item.
+            }
+          }
+          // {
+          //   // Percentage filter - activates a feature based on a percentage, to enable the feature flag for 50% of requests
+          //   name: "Microsoft.Percentage",
+          //   parameters: { Value: 50 }
+          // },
+          // { name: "FilterX" }, // Custom filter
+        ]
+      }
+    }
   };
 
   // Set the following environment variable or edit the value on the following line.
   const connectionString = process.env["APPCONFIG_CONNECTION_STRING"] || "<connection string>";
   const appConfigClient = new AppConfigurationClient(connectionString);
 
-  await cleanupSampleValues([featureFlag.key], appConfigClient);
+  await cleanupSampleValues([originalFeatureFlag.key], appConfigClient); // Ignore - cleanup
```

Comment:
cleanup?

Reply:
just added a comment to not confuse the reviewers during the design discussions.
Removed now.